index.bs

```diff
@@ -242,6 +229,14 @@ When the user agent is executing the <a>painting order</a>, it must populate the
 
 NOTE: A user agent might want to use a stack to efficiently compute the <a>set of owned text nodes</a> while implementing the <a>painting order</a>.
 
+Every {{Element}} has a list of <dfn>associated background image requests</dfn> which is initially an empty array.
+When the {{Element}}'s style requires a new image resource (to be displayed as background image), a new <a>image request</a> caused by the new resource is appended to the {{Element}}'s <a>associated background image requests</a>.
+Whenever an <a>image request</a> in an {{Element}}'s <a>associated background image requests</a> has become <a>completely available</a> and decoded, run the algorithm to <a>process an image that finished loading</a> with the {{Element}} and <a>image request</a> as inputs.
```

Comment:
Again, would be good to invoke this directly, but dunno where in the CSS case.

Also, let's drop the "decoded" bit.

Reply:
Dropped decoded